Makefile

```diff
@@ -207,10 +207,14 @@ CGO_ENABLED := 0
 endif
 export CGO_ENABLED
 
+export GIT_REPO 		 ?= github.com/operator-framework/operator-controller
+export VERSION           ?= $(shell git describe --tags --always --abbrev=0)
+export VERSION_PATH 	 ?= ${GIT_REPO}/internal/version
+export GIT_COMMIT 		 ?= $(shell git rev-parse HEAD)
```

Comment:
Looks like `debug.BuildInfo` contains the git commit with key `vcs.revision`. Could we use that instead of injecting it?